geode-core/src/main/java/org/apache/geode/internal/cache/BackupManager.java

```diff
@@ -569,6 +570,28 @@ private void createBackupDir(File backupDir) throws IOException {
     }
   }
 
+  private void backupOplog(File targetDir, Oplog oplog) throws IOException {
+    File crfFile = oplog.getCrfFile();
+    backupFile(targetDir, crfFile);
+
+    File drfFile = oplog.getDrfFile();
+    backupFile(targetDir, drfFile);
+
+    oplog.finishKrf();
+    File krfFile = oplog.getKrfFile();
+    backupFile(targetDir, krfFile);
+  }
+
+  private void backupFile(File targetDir, File file) throws IOException {
+    if (file != null && file.exists())
+      try {
+        Files.createLink(targetDir.toPath().resolve(file.getName()), file.toPath());
+      } catch (IOException | UnsupportedOperationException e) {
+        logger.warn("Unable to create hard link for + " + targetDir + ". Reverting to file copy");
```

Comment:
You have an extra `+` after inside the log statement quotes.

The Log4J2 way is to use their parameters:
```java
logger.warn("Unable to create hard link for {}. Reverting to file copy", targetDir);
```